src/Tribe/Repositories/Attendee/RSVP.php

```diff
@@ -104,4 +105,126 @@ public function checked_in_keys() {
 		return $this->limit_list( $this->key_name, parent::checked_in_keys() );
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	public function update_additional_data( $attendee, $ticket, $attendee_data ) {
+
+		$attendee_data = $this->format_attendee_data( $attendee, $ticket, $attendee_data );
+
+		try {
+			$query = $this->where( 'id', $attendee->ID )
+			              ->set( 'ticket_id', $ticket->ID )
```

Comment:
We should use set_args() I think here and pass the array into that so it's easier to work with $attendee_data instead of calling ->set() manually each time